x-pack/plugins/security_solution/public/resolver/data_access_layer/factory.ts

```diff
@@ -38,13 +37,6 @@ export function dataAccessLayerFactory(
       });
     },
 
-    /**
```

Comment:
The indices to use are now passed in to Resolver via props from the Security Solution app.